crates/egui/src/widgets/plot/mod.rs

```diff
@@ -437,6 +439,14 @@ impl Plot {
         self
     }
 
+    /// Whether to allow auto bounds. Default: `true`.
+    /// `false` keep bounds, it gives the ability with a pointer down to the drawn outside the plot area 
+    /// without ask auto bounds.
```

Comment:
This represents an example of use and does not describe the functionality.
I meant " If `false`, when the bounds is modified it will not be recalculated." but that doesn't seem necessary.